lab-papers-please/cpp-classification/src/main.cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+#include <fstream>
+#include "../include/nlohmann/json.hpp"
+#include <vector>
+#include <string>
+
+using json = nlohmann::json;
+using namespace std;
+
+class Universe {
+public:
```

Comment:
Forgot to mention, make it private, or add a TODO to make it private for future students, so they have a good example from the very beginning of hiding data.